components/src/core/AppBar/AppBar.tsx

```diff
@@ -26,7 +26,7 @@ const useUtilityClasses = (ownerState: AppBarProps): Record<AppBarClassKey, stri
 const Root = styled(MuiAppBar, {
     name: 'app-bar',
     slot: 'root',
-    shouldForwardProp: (prop) => prop !== 'backgroundImage',
+    shouldForwardProp: (prop) => !['backgroundImage', 'animationDuration'].includes(prop.toString()),
```

Comment:
If you're making changes in the src folder, we're going to need a changelog entry description what you've changed.   Same applies for EmptyState change.

Reply:
Done